.github/workflows/lint-and-test.yaml

```diff
@@ -68,7 +77,7 @@ jobs:
 
       - name: Prepare Environment
         run: |
-          yarn config set network-timeout 100000 -g
+          corepack enable
```

Comment:
having to do a `corepack enable` twice here is weird. hopefully this will not be necessary after my earlier comment

Reply:
I just started off with someone's tutorial saying that the `corepack enable` should be before `setup-node` action, but I guess the tutorial was just wrong.